samples/sample_mapper/mapper.py

```diff
@@ -1,5 +1,7 @@
 """A sample inline mapper app."""
 
+from __future__ import annotations
+
 from pathlib import PurePath
 from typing import Generator, List, Optional, Union
```

Comment:
Looks like some unused imports are left over. I'm not sure why flake8 isn't failing because of this.

Reply:
Good catch!

We are not using Pyflakes, so we don't get the `F401` error code anywhere. Probably worth including when we do #1242, so we even get pre-commit autofixes.

I've removed the lingering imports in `samples/` for now.